app/views/hyrax/my/_facet_limit.html.erb

```diff
@@ -1,9 +1,12 @@
   <ul class="dropdown-menu facet-values" aria-labelledby="<%= facet_field.field.parameterize %>">
     <% paginator = facet_paginator(facet_field, display_facet) %>
-    <%= render_facet_limit_list paginator, field_name %>
+    <%= render_facet_limit_list paginator, facet_field.key %>
```

Comment:
The changes in this file change the way we're pluralizing facet field labels, making use of Blacklight's `facet_field_label` helper which looks in i18n.

They also use Blacklight 6.9's new `more_html` key.